apis/pom.xml

```diff
@@ -193,19 +204,6 @@
   <profiles>
     <profile>
       <id>cassandra-40</id>
```

Comment:
kept this for backward compatibility.. in case any command uses `-P cassandra-40`, I don't want to break it..

Reply:
Yeah makes sense to me. Could add a comment explaining that it's left for backwards compatibility, used to be default.

Reply:
Comment added.